### ISSUE TITLE
Update projects using an out of date version.

### ISSUE THREAD

**Issue by codeviking** [allenai/varnish#205]
We'd like to update our projects to use the latest Varnish version where possible, so that they use the updated logo.

Here's a list, generated via `scalawag 0.1`:

- [ ] https://github.com/allenai/turkiq @allenai/varnish:^0.3.18
- [ ] https://github.com/allenai/leaderboard @allenai/varnish:^0.5.5
- [ ] https://github.com/allenai/bert-aristo-demo @allenai/varnish:^0.2.2
- [ ] https://github.com/allenai/s2-topics-skiff @allenai/varnish:^0.2.2
- [ ] https://github.com/allenai/s2-sanity @allenai/varnish:^0.1.6
- [ ] https://github.com/allenai/scispacy-demo @allenai/varnish:^0.3.19
- [ ] https://github.com/allenai/autocat @allenai/varnish:0.5.5
- [ ] https://github.com/allenai/hmasp @allenai/varnish:^0.2.2
- [ ] https://github.com/allenai/incubator-skiff @allenai/varnish:^0.3.18
- [ ] https://github.com/allenai/grover-demo @allenai/varnish:^0.3.16
- [ ] https://github.com/allenai/hack19-geoscholar @allenai/varnish:^0.3.18
- [ ] https://github.com/allenai/relation-generation @allenai/varnish:0.4.3
- [ ] https://github.com/allenai/gobo @allenai/varnish:^0.3.18
- [ ] https://github.com/allenai/leeskiffdemo @allenai/varnish:^0.2.2
- [ ] https://github.com/allenai/spike @allenai/varnish:^0.2.2
- [ ] https://github.com/allenai/ethical-turk @allenai/varnish:^0.3.18
- [ ] https://github.com/allenai/github-stats-2 @allenai/varnish:^0.1.6
- [ ] https://github.com/allenai/supp.ai @allenai/varnish:0.4.3
- [ ] https://github.com/allenai/aristo-demo-2019 @allenai/varnish:^0.3.16
- [ ] https://github.com/allenai/comet-demo @allenai/varnish:^0.2.2
- [ ] https://github.com/allenai/dolmsreason-demo @allenai/varnish:^0.3.18
- [ ] https://github.com/allenai/prior-demo @allenai/varnish:^0.5.5
- [ ] https://github.com/allenai/allennlp-demo @allenai/varnish:^0.3.19
- [ ] https://github.com/allenai/anli-demo @allenai/varnish:^0.2.2
- [ ] https://github.com/allenai/qasper-answers @allenai/varnish:0.5.5
- [ ] https://github.com/allenai/aimichal-yacht @allenai/varnish:^0.3.16
- [ ] https://github.com/allenai/hack19-prereqs-ui @allenai/varnish:^0.3.18

**Comment by aaronsarnat** [comment]
If we want to split this work among the 3 of us, I vote that we have me update the demos that are more recent and straightforward. One takeaway from the experience trying to update the comet demo is that legacy varnish apps require the big guns :)

**Comment by codeviking** [comment]
For now react router, here's a command that updates `varnish` and it's peers:

```
yarn add @allenai/varnish@^0.5.8 antd@^3.23.2 react@^16.9.0 @types/react@^16.9.0 react-dom@^16.9.0 @types/react-dom@^16.9.0
```

**Comment by codeviking** [comment]
For a project that uses `react-router`, you should also run:

```
yarn add --dev react-router-dom@^5.0.1 
yarn add react-router@^5.0.1 react-router-dom@^5.0.1
yarn add --dev @types/react-router@^5.0.1 @types/react-router-dom@^5.0.1
```

**Comment by codeviking** [comment]
For projects that have a favicon statically in `ui/public`:

```
cd ui/public
rm favicon.ico
wget https://allenai.org/favicon.ico
```

**Comment by aaronsarnat** [comment]
Thanks @codeviking! I see you're planning to do leaderboard. FYI, I have a PR for the non varnish asset updates: https://github.com/allenai/leaderboard/pull/716

It was approved, but I haven't merged yet since I thought it should happen around the same time as the micro-header update.

**Comment by codeviking** [comment]
@aaronsarnat Sure, I'll just let you take care of it then.

**Comment by codeviking** [comment]
We likely should've taken care of #176 prior to starting this ticket.

Let's go ahead and pause on this ticket until we resolve #176, at which point we'll need to bump the projects we've already done to `0.5.9` (which should be very straightforward, given it should be a minimal release.

**Comment by aaronsarnat** [comment]
@codeviking should i wait to update leaderboard and allennlp until after https://github.com/allenai/varnish/pull/210 gets released into varnish?

**Comment by aaronsarnat** [comment]
Also, thanks for putting this list together

**Comment by codeviking** [comment]
> @codeviking should i wait to update leaderboard and allennlp until after #210 gets released into varnish?

No, let's proceed.  Logo updates > legal name updates, and legal stuff might take time.  Sorry for the noise.

**Comment by jonborchardt** [comment]
keeping the ticket, even though we should re run scalawag
